packages/uicore/src/components/Avatar/Avatar.stories.tsx

```diff
@@ -74,3 +74,10 @@ export const Image: Story<AvatarProps> = args => (
     />
   </Layout.Horizontal>
 )
+
+// Edge case for testing, this should not render anything
+export const NullName: Story<AvatarProps> = args => (
```

Comment:
I was thinking should not we use 'Not Available' when we do not have data like these?

Reply:
Good question. Let me direct this to Kapil Digani.